azureadps-2.0-preview/AzureAD/Get-AzureADDevice.yml

```diff
@@ -31,8 +31,8 @@ examples:
 
     ObjectId                             DeviceId                             DisplayName
     --------                             --------                             -----------
-    3cb87a8f-0a41-4ca8-8910-e56cc00114a3 48445467-033c-42ca-8e38-8d181db1d49c bastias_WindowsPhone_5/1/2016_12:53 PM
-    62aae804-8b1a-4ab7-8fda-5068aed1a1f7 3cb87a8f-0a41-4ca8-8910-e56cc00114a3 New Device
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb 48445467-033c-42ca-8e38-8d181db1d49c bastias_WindowsPhone_5/1/2016_12:53 PM
+    62aae804-8b1a-4ab7-8fda-5068aed1a1f7 aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb New Device
     d4fe7726-5966-431c-b3b8-cddc8fdb717d 293872f6-c006-4e6a-8629-07847c5ab078 New Device
```

Comment:
Not all ObjectIds have been updated.

Reply:
These may have to be manual changes based on current capabilities.